brion/morphology.cpp

```diff
@@ -69,27 +69,27 @@ CellFamily Morphology::getCellFamily() const
     return _impl->plugin->getCellFamily();
 }
 
-Vector4fsPtr Morphology::readPoints() const
+Vector4fsPtr Morphology::readPoints()
```

Comment:
Do we really need to break the API here. The inconsistency in the current code is in the SWC reader, which actually returns private data and makes it shared. The H5 reader is reading the data each time the function is called. If SWC is modified to return a copy there wouldn't be any logical issue with this const. Why it must be different in the ZeroEQ case despite the function returns always the same value (as long as this function doesn't really have visible side effects on the object state it should be considered const).